config/sewing-kit/index.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  Package,
+  createComposedProjectPlugin,
+  createProjectTestPlugin,
+  createProjectBuildPlugin,
+} from '@sewing-kit/plugins';
+import {react} from '@sewing-kit/plugin-react';
+import {javascript} from '@sewing-kit/plugin-javascript';
+import {typescript} from '@sewing-kit/plugin-typescript';
+import {buildFlexibleOutputs} from '@sewing-kit/plugin-package-flexible-outputs';
+
+function addLegacyDecoratorSupport(config) {
+  return {
+    presets: config.presets,
+    plugins: (config.plugins || []).map(plugin => {
+      const pluginName = Array.isArray(plugin) ? plugin[0] : plugin;
+
+      if (pluginName.includes('@babel/plugin-proposal-decorators')) {
+        return [pluginName, {legacy: true}];
+      } else if (
+        pluginName.includes('@babel/plugin-proposal-class-properties')
+      ) {
+        return [pluginName, {loose: true}];
+      } else {
+        return plugin;
+      }
+    }),
+  };
+}
+
+export function quiltPackage({binaryOnly = true, jestEnv = 'jsdom'} = {}) {
```

Comment:
What's this `binaryOnly` flag for? It's defaulted to true and I don't see it being used anywhere else?

Reply:
I think I just copied that from Chris' [Quilt config](https://github.com/lemonmade/quilt/blob/main/config/sewing-kit/index.ts). Wasn't sure if we wanted `.mjs` and `.esnext` files in our builds. I can get rid of it and hardcode that to be `true` instead for now.

Reply:
Let's leave them out for now. Adding support for `.mjs` and `.esnext` feels out of scope in validating sewing-kit-next in quilt.

Reply:
disagree

Reply:
I stand corrected. @b-ryu, sorry for the confusion. We'll need this.